controller/search_blackbox_test.go

```diff
@@ -575,6 +577,19 @@ func (s *searchBlackBoxTest) TestSearchQueryScenarioDriven() {
 
 	wirepo := workitem.NewWorkItemRepository(s.DB)
 
+	labelNames := []string{"IMPORTANT", "backend", "UI", "REST"}
+	testFixtures, err := tf.NewFixture(s.DB, tf.Labels(4, func(f *tf.TestFixture, idx int) error {
+		f.Labels[idx].Name = labelNames[idx]
+		return nil
+	}))
+	require.Nil(s.T(), err)
+	require.Len(s.T(), testFixtures.Labels, 4) // not required but being explicit
+	// following assigment makes the test more readable
+	lblImportant := testFixtures.Labels[0]
+	lblBackend := testFixtures.Labels[1]
+	lblUI := testFixtures.Labels[2]
+	lblREST := testFixtures.Labels[3]
+
 	// create 3 WI with state "resolved" and iteration 1
```

Comment:
can't you use the TextFixture to generate those 3 work items ?

Reply:
I can and I will while I refactor related tests later :) I kept the older code as it is, But in new code, I am using `TestFixture`. If I use `TestFixture` in that place, need to replace all its usages below and there are few other scenarios where I can use `TestFixtures` (it is a lonngg test).
Is it okay?

Reply:
@pranavgore09 yes, just add 

```go
tf.WorkItems(3, func(fxt *tf.TestFixture, idx int)error{
  fxt.WorkItems[idx].Fields[workitem.SystemState] = workitem.SystemStateResolved
  fxt.WorkItems[idx].Fields[workitem.SystemIteration] = sprint1.ID.String()
  return nil
})
```
to the creation of the test fixture above.

Reply:
okay @kwk 
Sorry @xcoulon I did misread, actually I was not referring to WorkItems my in test once they are created. Now fully cleaned up test looks like this https://github.com/fabric8-services/fabric8-wit/pull/1640/commits/46a3fe097d1ef83dfab077182e2ed21aee75c515